test/asm/multi-line-strings.asm

```diff
@@ -21,7 +21,8 @@ printarg: MACRO
 ENDM
 
 	printarg "
-	printarg """
+	printarg """multi-line
+arg"""
```

Comment:
Tiny nitpick: given that the macro prepends "arg", I find the test output slightly confusing, and "argument" would be ever so slightly clearer.